src/components/input-field/input-field.types.ts

```diff
@@ -11,5 +11,4 @@ export type InputType =
     | 'textarea'
     | 'time'
     | 'url'
-    | 'urlAsText'
```

Comment:
Don't merge this yet. It's currently a breaking change, and I want to make it non-breaking.